holoviews/plotting/util.py

```diff
@@ -216,16 +216,17 @@ def split_dmap_overlay(obj, depth=0):
     """
     layers = []
     if isinstance(obj, DynamicMap):
-        if issubclass(obj.type, NdOverlay) and not depth:
-            for v in obj.last.values():
-                layers.append(obj)
-        elif issubclass(obj.type, Overlay):
-            if obj.callback.inputs and is_dynamic_overlay(obj):
-                for inp in obj.callback.inputs:
-                    layers += split_dmap_overlay(inp, depth+1)
-            else:
+        if obj.type is not None:
```

Comment:
I had looked at this at some point and didn't understand why ``obj.type`` would ever be None at this point. Do you have any idea?

Reply:
I assumed it was because it is a DynamicMap and not yet computed or something, but I honestly have no idea. The depth was 2 on that if that provides any insight.